docs/dns/rrsets.rst

```diff
@@ -213,19 +213,19 @@ https://www.huque.com/bin/gen_tlsa.  We are planning to provide a tool that is
 connected directly to our API in the future.  For full detail on how ``TLSA``
 records work, please refer to RFC 6698.
 
-The ``TLSA`` sha256 hash can also be generated with that command::
+The ``TLSA`` sha256 hash can also be generated with the following command::
 
-    openssl x509 -in /etc/letsencrypt/live/${name}/fullchain.pem -noout -pubkey \
+    openssl x509 -in </path/to/fullchain.pem> -noout -pubkey \
     | openssl pkey -pubin -outform DER \
     | openssl sha256
 
-**Pro tip:** To put only the hash into an environment variable can this comand
+**Pro tip:** To put only the hash into an environment variable can this command
```

Comment:
Sorry, still not quite right: `can this command be used` implies a question.  `this command can be used` is what you want.